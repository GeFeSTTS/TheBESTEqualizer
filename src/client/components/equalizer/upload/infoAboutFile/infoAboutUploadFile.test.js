import React from 'react';
import { shallow } from 'enzyme';

import InfoAboutTrack from './infoAboutUploadFile';

describe('test infoAboutUploadFile component', () => {
  const props = {
    trackname: 'fakeName',
  };

  let wrapper;
  beforeEach(() => {
    wrapper = shallow(<InfoAboutTrack {...props} />);
  });

  it('should render element with `trackname`', () => {
    expect(wrapper.find('div').text()).toEqual('<FontAwesomeIcon />Track name: fakeName');
  });

  it('should render component properly', () => {
    expect(wrapper.debug()).toMatchSnapshot();
  });
});
