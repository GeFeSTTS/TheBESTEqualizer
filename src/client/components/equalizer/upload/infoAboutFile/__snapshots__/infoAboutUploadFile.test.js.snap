// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test infoAboutUploadFile component should render component properly 1`] = `
"<div id=\\"detailstrack\\">
  <FontAwesomeIcon icon={{...}} className=\\"icon\\" border={false} mask={{...}} fixedWidth={false} inverse={false} flip={{...}} listItem={false} pull={{...}} pulse={false} rotation={{...}} size={{...}} spin={false} symbol={false} title=\\"\\" transform={{...}} />
  <span className=\\"TrackName\\">
    Track name:
  </span>
   
  fakeName
</div>"
`;
